Twig/Extension/MediaExtension.php

```diff
@@ -169,7 +169,7 @@ public function thumbnail($media = null, $format, $options = array())
 
         $options = array_merge($defaultOptions, $options);
 
-        $options['src'] = $provider->generatePublicUrl($media, $format);
+        $options = $provider->getHelperProperties($media, $format, $options);
```

Comment:
Why was this merged? It should have raised some questions IMO @core23 @OskarStark . It's already in 3 releases now, so we can't revert it can we? How can we fix this? Please have a look a #1065